usaspending_api/api_contracts/contracts/v2/idvs/awards.md

```diff
@@ -68,6 +68,8 @@ List child IDVs, child awards, or grandchild awards for IDV
                         "awarding_agency": "DEPARTMENT OF THE INTERIOR (DOI)",
                         "funding_agency_id": 228,
                         "awarding_agency_id": 228,
+                        "funding_agency_slug": "general-services-administration-gsa",
```

Comment:
In this example the `*_slug` does not match the Agency Name. 

Also, in all of these example responses can you remove the abbreviation from the name and slug? Makes sense that the sample slug would have it since the sample name has it, but since the actual response isn't formatted that way neither one should contain the abbreviation.

Reply:
Fixed the slug names and removed the abbreviation